src/java/org/apache/cassandra/streaming/StreamingState.java

```diff
@@ -70,6 +71,14 @@ public class StreamingState implements StreamEventHandler
     // API for state changes
     public final Phase phase = new Phase();
 
+    @Override
+    public long unsharedHeapSize()
+    {
+        long costOfPeers = peers().size() * (ObjectSizes.IPV6_SOCKET_ADDRESS_SIZE + 48); // 48 represents the datastructure cost computed by the JOL
```

Comment:
I don't like the 48 magic constant, but understand why it's hard to generate from the wrapped map/set.